app/models/user.rb

```diff
@@ -26,4 +27,12 @@ def service_owner?
   def delivery_partner?
     !service_owner?
   end
+
+  private
+
+  def email_cannot_be_changed_after_create
+    if email_changed?
+      errors.add(:email, "cannot be changed once a user has been created")
```

Comment:
This should probably be I18n'd (NB: Use `errors.add(:email, :blah)` here rather than calling I18n.t directly)

Reply:
Good call 👍